test/metabase/api/user_test.clj

```diff
@@ -215,7 +215,35 @@
                   (filter mt/test-user?)
                   group-ids->sets
                   mt/boolean-ids-and-timestamps
-                  (map #(dissoc % :is_qbnewb :last_login))))))))
+                  (map #(dissoc % :is_qbnewb :last_login)))))))
+  (testing "GET /api/user?include_deactivated=false should return only active users"
+    (is (= (->> [{:email                  "crowberto@metabase.com"
```

Comment:
Since we only care about _which_ users are returned, not the rest of the data, you can use `partial=` to assert on a just a subset of the data in the results.

It's not used by the other tests since it was added pretty recently

Reply:
Good to know! but to keep things simpler I've changed to only check the email.